integration-tests/src/test/java/com/redhat/parodos/integration/prebuilt/RestTest.java

```diff
@@ -0,0 +1,59 @@
+package com.redhat.parodos.integration.prebuilt;
+
+import java.util.Arrays;
+
+import com.redhat.parodos.flows.common.WorkFlowTestBuilder;
+import com.redhat.parodos.sdk.api.WorkflowApi;
+import com.redhat.parodos.sdk.invoker.ApiException;
+import com.redhat.parodos.sdk.model.ArgumentRequestDTO;
+import com.redhat.parodos.sdk.model.WorkFlowExecutionResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowRequestDTO;
+import com.redhat.parodos.sdk.model.WorkFlowStatusResponseDTO;
+import com.redhat.parodos.sdk.model.WorkRequestDTO;
+import com.redhat.parodos.sdkutils.WorkFlowServiceUtils;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class RestTest {
+
+	private final String workflowName = "restWorkFlow";
+
+	private final String taskName = "restTask";
+
+	@Test
+	public void runFlow() throws InterruptedException, ApiException {
+		WorkFlowTestBuilder.TestComponents components = new WorkFlowTestBuilder().withDefaultProject()
+				.withWorkFlowDefinition(workflowName).build();
+
+		// EXECUTE WORKFLOW
+		WorkflowApi workflowApi = new WorkflowApi(components.apiClient());
+
+		// Define WorkRequests
+		String url = "http://workflow-service:8080/api/v1/workflowdefinitions";
+		WorkRequestDTO workGet = new WorkRequestDTO().workName(taskName)
+				.arguments(Arrays.asList(new ArgumentRequestDTO().key("url").value(url),
+						new ArgumentRequestDTO().key("method").value("get"),
+						new ArgumentRequestDTO().key("username").value("test"),
+						new ArgumentRequestDTO().key("password").value("test")));
+
+		WorkFlowRequestDTO workFlowRequestGet = new WorkFlowRequestDTO().projectId(components.project().getId())
+				.workFlowName(workflowName).works(Arrays.asList(workGet));
+
+		WorkFlowExecutionResponseDTO workFlowResponseDTO = workflowApi.execute(workFlowRequestGet);
+
+		assertNotNull(workFlowResponseDTO.getWorkFlowExecutionId());
+		assertNotNull(workFlowResponseDTO.getWorkStatus());
+		assertEquals(WorkFlowExecutionResponseDTO.WorkStatusEnum.IN_PROGRESS, workFlowResponseDTO.getWorkStatus());
+
+		WorkFlowStatusResponseDTO workFlowStatusResponseDTO = WorkFlowServiceUtils.waitWorkflowStatusAsync(workflowApi,
+				workFlowResponseDTO.getWorkFlowExecutionId());
+
+		assertNotNull(workFlowStatusResponseDTO);
+		assertNotNull(workFlowStatusResponseDTO.getWorkFlowExecutionId());
+		assertNotNull(workFlowStatusResponseDTO.getStatus());
+		assertEquals(WorkFlowStatusResponseDTO.StatusEnum.COMPLETED, workFlowStatusResponseDTO.getStatus());
```

Comment:
According to the project rules, we should use `assertThat().isNotNull`, `assertThat().isEqualTo`, etc.

Reply:
this piece of code was copied from existing integration tests. so these rules probably do not apply there